multidict/_multidict.c

```diff
@@ -863,6 +864,22 @@ multidict_class_getitem(PyObject *self, PyObject *arg)
     return self;
 }
 
+
+PyDoc_STRVAR(sizeof__doc__,
+"D.__sizeof__() -> size of D in memory, in bytes");
+
+static inline PyObject *
+_multidict_sizeof(PyObject *self)
```

Comment:
Mb more correct `_multidict_sizeof(PyObject *self, PyObject *args)` according to `PyCFunction` and `PyMethodDef` ?

Reply:
We use `METH_O` convention in other places.
I've updated the PR to accept `MultiDictObject * self` as an argument.